server/server.go

```diff
@@ -129,7 +129,18 @@ func NewServer(logger hclog.Logger, config *Config) (*Server, error) {
 			Blockchain: m.blockchain,
 		}
 		// start transaction pool
-		m.txpool, err = txpool.NewTxPool(logger, m.config.Seal, m.config.Locals, m.config.NoLocals, m.config.PriceLimit, m.config.MaxSlots, m.chain.Params.Forks.At(0), hub, m.grpcServer, m.network)
+		m.txpool, err = txpool.NewTxPool(
+			logger,
+			m.config.Seal,
+			m.config.Locals,
+			m.config.NoLocals,
+			m.config.PriceLimit,
+			m.config.MaxSlots,
+			m.config.SpeedUpMin,
+			m.chain.Params.Forks.At(0),
+			hub,
+			m.grpcServer,
+			m.network)
 		if err != nil {
```

Comment:
I think it's time we introduced a `TxPoolParams` in the `txpool` package, and just made the `NewTxPool` method accept those params as a single argument.

It's getting ridiculous with these 3000 arguments

Reply:
I feel the same way about it. A `TxPoolConfig` should be introduced.

Perhaps that could go as a separate PR ?

Reply:
I discussed this with @munna0908  regarding PR #216, we decided to leave it for now, and thing of a better solution down the road 👍